content/node-runners/setup/docker.md

```diff
@@ -114,38 +62,23 @@ When the installation finishes, Docker starts automatically. The "whale" icon in
 #### For WINDOWS users: Open a command-line terminal and type the following command:
 
 ```bash
-docker run --cap-add NET_ADMIN -p 4449:4449 --name myst -v $YOUR_MYSTERIUM_DIR:/var/lib/mysterium-node mysteriumnetwork/myst:latest service --agreed-terms-and-conditions wireguard
+docker run --cap-add NET_ADMIN -d -p 4449:4449 --name myst -v myst-data:/var/lib/mysterium-node mysteriumnetwork/myst:latest service --agreed-terms-and-conditions
```

Comment:
Add `-d` option by default and you will not need to have an extra note about it.